helper/ResponseVariableFormatter.php

```diff
@@ -139,13 +139,16 @@ public static function formatVariableToPci(ResponseVariable $var)
      *
      * @param array $testResultVariables - the array output from oat\taoOutcomeUi\model\ResultsService::getStructuredVariables();
      * @param array $itemFilter = [] - the array of item uri to be included in the formatted output, all item if empty.
+     * @param bool $rawFile = false - when true, it formats the embedded file details, when false, it returns the file reference (uri)
      * @return array
      * @throws \common_Exception
      */
     public static function formatStructuredVariablesToItemState(
         array $testResultVariables,
-        array $itemFilter = []
+        array $itemFilter = [],
+        bool $rawFile = false
```

Comment:
A flag changing a behavior or, like in this case, a response contract, sorta smells to be fair.
The change would probably work a bit better if a new field was added into a file-typed `response`. But I guess that would break the integration flow with the other extensions. So, I don't really have a good suggestion here.

Reply:
Indeed, thanks a lot for your comment. It made me rethink and turned out that the other places also support the new format which is the expected one for an ItemState. I have removed the flag ;)